framework/typescript/client.mdx

```diff
@@ -41,10 +41,18 @@ When your service is running in development mode `process.env.NODE_ENV=developme
 ## Code Example
 
 ```typescript
-import { Client } from "@novu/framework";
-
-const client = new Client({
-  secretKey: "my-secret-key",
-  strictAuthentication: false,
+import { Client as NovuFrameworkClient } from "@novu/framework";
+import { serve } from "@novu/framework/next";
+import { passwordResetWorkflow } from "./workflows";
+
+export const { GET, POST, OPTIONS } = serve({
+  client: new NovuFrameworkClient({
+    secretKey: process.env.NOVU_SECRET_KEY || "ACTUAL_NOVU_SECRET_KEY_VALUE",
```

Comment:
Let's keep it as follows so as not to prompt users to hardcode secrets in their code.

```suggestion
    secretKey: process.env.NOVU_SECRET_KEY,
```